scraper_common.js

```diff
@@ -75,7 +75,16 @@ async function execute(usePuppeteer, scrapers) {
                 startTime,
                 numberAppointments
             );
+
+            // Add the current timestamp to each individual location
+            const nowTimestamp = moment().format();
+            returnValue?.individualLocationData.map((loc) => {
+                loc.timestamp = nowTimestamp;
```

Comment:
since we have this at the scraper run (parent location) level I think we should use that instead. So loc.timestamp = returnValue.timestamp

At least I think that's where we're currently passing in timestamp for everything?

Reply:
Much better!